test/integration/simulacron-helper.js

```diff
@@ -0,0 +1,199 @@
+var Client = require('node-rest-client').Client;
```

Comment:
I have chosen an high level library, because the built-in module is fairly low-level, which would need to append the data (received in blocks on 'data' event) response and listing to 'end' event to parse the response in all requests.
`
const https = require('https');
 
https.get('url', (resp) => {
  let data = '';
 
  // appending data
  resp.on('data', (chunk) => {
    data += chunk;
  });
 
  // Wait to finish to parse JSON
  resp.on('end', () => {
    JSON.parse(data);
  });
 
}).on("error", (err) => {
  console.log("Error: " + err.message);
});
`

Reply:
I would prefer to maintain a utility function against a core module than a dependency to a external module.